meinberlin/assets/scss/components_user_facing/_phase_info.scss

```diff
@@ -1,16 +1,6 @@
 .phase-info {
     background-color: $blue-dark;
     color: $white;
-
```

Comment:
this was introduced to fix the phase-info for single module projects, so removing it will break them again. I think we need to change this so it doesn't apply to the phase-info in multi-module projects. Not sure if that is the best way but having a separate `.tabpanel .phase-info {` seems to work for me.

Reply:
sure, locally there is no single module project by default, so you have to create one, rough steps are:

1.  go to http://127.0.0.1:8003/dashboard/organisations/liqd/projects/
2. click on "Add new project", add the project and then fill all the required fields
3. add a module (e.g. participatory budgeting 2) and fill in all fields
4. click add module to project
5. publish project

If you can't reproduce it we can also do a quick call :)